packages/app/src/app/components/Preview/DevTools/Terminal/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Terminal } from 'xterm';
+
+export type TerminalWithFit = Terminal & {
+  fit(): void;
+};
+
+export type ShellT = {
+  id: string;
+  title: string;
+  script: string | undefined;
```

Comment:
@CompuIves This can just be

```suggestion
  script?: string;
```

Reply:
This breaks some types, because I can't refer to script anymore as `shell.script` without doing `if 'script' in shell`.

Reply:
Hmmm that's weird...

`script?: string` means "it's a string, but it can happen it's not there (so `undefined`).

Reply:
I think the difference can be seen with this for example: 
![image](https://user-images.githubusercontent.com/587016/60801331-21195800-a177-11e9-935e-0ae2a53edcf2.png)